testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -928,6 +928,158 @@ pub fn boot_to_epoch_3(
     info!("Bootstrapped to Epoch-3.0 boundary, Epoch2x miner should stop");
 }
 
+///
+/// * `stacker_sks` - must be a private key for sending a large `stack-stx` transaction in order
+///   for pox-4 to activate
+pub fn boot_to_epoch_3_flash_blocks(
```

Comment:
Can you update this comment and function name to be more clear about what it is doing? This is basically the same as `boot_to_epoch_3` except that it stops at epoch 3 height - 2, right?